### ISSUE TITLE
Spyder switches to plot tab when debugging in console

### ISSUE THREAD

**Issue by Caseyb87** [spyder-ide/spyder#15705]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
The Spyder UI opens the "Plots" tab when actively debugging code in the console, regardless of which tab is currently being used.

### What steps reproduce the problem?

1. Debug the following code (set break point at c=1):

```python
import matplotlib.pyplot as plt
import numpy as np

a=np.arange(0,10)
b=np.arange(0,10)

plt.plot(a,b)

c=1
```
2. Allow code to run and stop at break point
3. Either leave current tab as "Help" (or open "Variable Explorer"/"Files").
4. type c=1 into console

### What is the expected output? What do you see instead?

The expected behaviour would be to stay on the current tab. What happens instead is the "Plots" tab opens. 

When debugging code, I am most often interested in using the "Variable Explorer" tab. If I have this open and then start changing some variables in the console, I don't expect Spyder to yank me out of "Variable Explorer" and into "Plots".

As a side note, I also get the:

> Figures now render in the Plots pane by default. To make them also appear inline in the Console, uncheck "Mute Inline Plotting" under the Plots pane options menu. 

print out only after typing in the console. If this really needs to be shown, then shouldn't this happen before I type something into the console? Is this perhaps why Spyder is rudely switching tabs on me?

## Versions

* Spyder version: 5.0.3
* Python version: 3.8.5 64-bit
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: Windows 10

**Comment by ccordoba12** [comment]
Hey @Caseyb87, thanks for reporting. Did you deactivated the option called `Mute inline plotting` in the Options menu of Plots?

![imagen](https://user-images.githubusercontent.com/365293/119688430-98107600-be0d-11eb-9bf4-beb98ae527e6.png)

That's what avoids automatically showing that pane after each plot.

**Comment by Caseyb87** [comment]
No, that option was enabled. 

(Otherwise I don't think the "Figures now render in the Plots pane by default. To make them also appear inline in the Console, uncheck "Mute Inline Plotting" under the Plots pane options menu." text would show - it would show the plot in the console instead)

**Comment by ccordoba12** [comment]
I verified that this is the case if you're not debugging as well, which I find very annoying. I'll fix it in our 5.1.0 version, to be released in July.
